<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelect</class>
 <widget class="QWidget" name="ColorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorSelect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="3">
    <widget class="QWidget" name="displayWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="greenSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="redSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="blueSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="redSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="greenSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="blueSpinBox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
